sbot/metadata.py

```diff
@@ -61,7 +65,10 @@ def _find_file(search_path: str) -> Optional[str]:
 
 def _read_file(path: str) -> Dict[str, Any]:
     with open(path) as file:
-        obj = json.load(file)
+        try:
+            obj = json.load(file)
+        except json.decoder.JSONDecodeError:
+            raise RuntimeError("Unable to decode metadata. Ask a volunteer for help.")
```

Comment:
I'm not sure I'm a fan of saying things like "Ask a volunteer for help." in error messages. However, I would be in favour of installing a global error handler that prints some brief debugging advice at the end of the traceback, including something along the lines of "if all else fails ask a volunteer for help".

Reply:
I deliberately added it here because bad metadata is unlikely to be their fault. 

I'm ok with it in `sbot` only.